packages/trpc/server/routers/viewer/webhook/list.handler.ts

```diff
@@ -17,11 +17,23 @@ export const listHandler = async ({ ctx, input }: ListOptions) => {
     /* Don't mixup zapier webhooks with normal ones */
     AND: [{ appId: !input?.appId ? null : input.appId }],
   };
+
+  const user = await prisma.user.findFirst({
+    where: {
+      id: ctx.user.id,
+    },
+    select: {
+      teams: true,
+    },
+  });
+
   if (Array.isArray(where.AND)) {
     if (input?.eventTypeId) {
       where.AND?.push({ eventTypeId: input.eventTypeId });
     } else {
-      where.AND?.push({ userId: ctx.user.id });
+      where.AND?.push({
```

Comment:
We want to list all workflows here, including team workflows. This is only used to check if the subscriber URL already exists (with `subscriberUrlReserved`)